oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
Well I think the question of whether or not `InvalidXsrfTokenError` should continue to exist has not yet been answered. It is a module-scope module-public code element, but it is not raised by any part of the public API of the module. The smell I'm getting is that it should be either privatized or dropped. @jonparrott, am I making sense?